src/libraries/System.Private.CoreLib/src/System/Diagnostics/DebuggerDisableUserUnhandledExceptionsAttribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Diagnostics
+{
+    /// <summary>
+    /// If a .NET Debugger is attached which supports the Debugger.BreakForUserUnhandledException(Exception) API,
+    /// this attribute will prevent the debugger from breaking on user-unhandled exceptions when the
+    /// exception is caught by a method with this attribute, unless BreakForUserUnhandledException is called.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method)]
+    public sealed class DebuggerDisableUserUnhandledExceptionsAttribute : Attribute
```

Comment:
It probably isn't going to make much of a difference, but should we add this new attribute to

https://github.com/dotnet/runtime/blob/d710379da9b2699bba0a8f26d65d3a6e8549f26a/src/libraries/System.Private.CoreLib/src/ILLink/ILLink.LinkAttributes.Shared.xml#L2-L31

That way it gets trimmed when DebuggerSupport is off.